js/modules/k6/experimental/fs/errors.go

```diff
@@ -0,0 +1,71 @@
+package fs
+
+// newFsError creates a new Error object of the provided kind and with the
+// provided message.
+func newFsError(k errorKind, message string) *fsError {
+	return &fsError{
+		Name:    k.String(),
```

Comment:
Out of curiosity, why do we need two properties with the same data? If the name is calculated from the kind, why can't we obtain this dynamically?

Reply:
The reason for having an explicit `Name` field here is that it will be exposed to the Javascript runtime directly as the `.name` property of errors returned by `fs` operations. This is a convenience to enforce a certain "form" to errors exposed to users I replicated from the `webcrypto` project.

Whereas we use the `kind` attribute on the Go side.

Reply:
Okay, I see :+1: 

Just sharing that I might think that it's generally possible to achieve without the duplication by using https://pkg.go.dev/github.com/dop251/goja#Object.DefineAccessorProperty

But I guess in that case, it requires many changes for generating, and effort won't cost the result :+1: